test/test_files/ldbc/ldbc-interactive/interactive-short.test

```diff
@@ -0,0 +1,46 @@
+-GROUP LDBCTest
+-DATASET ldbc-sf01
+-BUFFER_POOL_SIZE 1073741824
+
+--
+
+-CASE LDBCInteractiveShort
```

Comment:
For IS2 & 6,

can u replace `replyOf*` with a bounded recursive pattern e.g. `replyOf*1..2`. Add document how changed the query. I think we won't be able to support Kleene star for a long time. Meanwhile it will be good to have some similar query running.